spec/unit/ssl/configuration_spec.rb

```diff
@@ -1,141 +0,0 @@
-#! /usr/bin/env ruby
-#
-
-require 'spec_helper'
-require 'puppet/ssl/configuration'
-
-describe Puppet::SSL::Configuration do
-  let(:localcacert) { "/path/to/certs/ca.pem" }
-
-  let(:ssl_server_ca_auth) { "/path/to/certs/ssl_server_ca_auth.pem" }
-
-  # different UTF-8 widths
-  # 1-byte A
-  # 2-byte ۿ - http://www.fileformat.info/info/unicode/char/06ff/index.htm - 0xDB 0xBF / 219 191
-  # 3-byte ᚠ - http://www.fileformat.info/info/unicode/char/16A0/index.htm - 0xE1 0x9A 0xA0 / 225 154 160
-  # 4-byte 𠜎 - http://www.fileformat.info/info/unicode/char/2070E/index.htm - 0xF0 0xA0 0x9C 0x8E / 240 160 156 142
-  let (:mixed_utf8) { "A\u06FF\u16A0\u{2070E}" } # Aۿᚠ𠜎
```

Comment:
One thing that is tested in this spec implicitly is the usage of UTF-8 in the certificates; I think this setup with UTF-8 in the certficates should be moved to the validator spec? @justinstoller does this have any implications for the `#create_chained_pki` method from #6797 ?

Reply:
The chained pki method doesn't create either CRL or Cert bundles with UTF8. I can update it to do so.

If chained pki is updated to create bundles with UTF8 characters then I think as long as the bundles are loaded into OpenSSL::X509::* objects correctly we're getting the testing that we need. And I assume that those tests will be with whatever utility module we write.

Reply:
Updated my PR

Reply:
Cool; do you think that we should wait for the PR to be merged, and then get this PR to use the same pki test setup? Or merge this in as is, and refactor the tests separately?

Reply:
Either way, if the other one goes in first, lets update this, but lets not block on this PR on that.